ember-flight-icons/tests/acceptance/index-test.js

```diff
@@ -11,6 +11,6 @@ module('Acceptance | icon index', function (hooks) {
     await visit('/');
     // added this because the icons are not rendering as quickly
     await waitFor('.flight-icon', { timeout: 1000 });
-    assert.dom('[data-test-target="icon-grid"] [data-test-icon]').exists();
+    assert.dom('[data-test-target="icon-grid"]').exists();
```

Comment:
I think we could test this in Percy instead in the future, cause (I think) technically the `ul` could be there with no icons displaying https://github.com/hashicorp/flight/issues/93

Reply:
https://github.com/hashicorp/flight/pull/211

Reply:
This change more or less makes this test not worth having imo as the point of the test was to confirm the sprite loaded _at all_ and this is just confirming the page has loaded.

Reply:
i was confused by this test TBH, cause it wasn't that way in the DOM with the icons displaying? i can add it back, but maybe a named hook on the icon itself makes more sense, in addition to the percy check? how about i make a PR to propose what i mean